# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gsmlib/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gsmlib-1.10/gsmlib/gsm_me_ta.h.vanilla	2006-03-04 19:22:37.360131000 +0100
+++ gsmlib-1.10/gsmlib/gsm_me_ta.h	2006-03-04 19:23:37.311877750 +0100
@@ -291,8 +291,8 @@
     // 3 disable phone receive RF circuits only
     // 4 disable phone both transmit and receive RF circuits
     // 5...127 implementation-defined
-    int MeTa::getFunctionalityLevel() throw(GsmException);
-    void MeTa::setFunctionalityLevel(int level) throw(GsmException);
+    int getFunctionalityLevel() throw(GsmException);
+    void setFunctionalityLevel(int level) throw(GsmException);
 
     // return battery charge status (+CBC):
     // 0 ME is powered by the battery
@@ -386,13 +386,13 @@
     void setCallWaitingLockStatus(FacilityClass cl,
                                   bool lock)throw(GsmException);
 
-    void MeTa::setCLIRPresentation(bool enable) throw(GsmException);
+    void setCLIRPresentation(bool enable) throw(GsmException);
     //(+CLIR)
     
     // 0:according to the subscription of the CLIR service
     // 1:CLIR invocation
     // 2:CLIR suppression
-    int MeTa::getCLIRPresentation() throw(GsmException);
+    int getCLIRPresentation() throw(GsmException);
 
     friend class Phonebook;
     friend class SMSStore;
